tests/ExporterTest.php

```diff
@@ -187,16 +187,46 @@ public function testGetProductsException()
         $exporterMock->getProducts();
     }
 
+    public function providerCreateProductItem()
+    {
+        return array(
+            array(1, 2, 3, 4, 3, 4),
+            array(1, 2, false, false, 1, 2)
+        );
+    }
+
     /**
      * The tested method should return instance of \Findologic\Plentymarkets\Product
+     *
+     * @dataProvider providerCreateProductItem
      */
-    public function testCreateProductItem()
+    public function testCreateProductItem($priceId, $rrpId, $configPriceId, $configRrpId, $expectedPriceId, $expectedRrpId)
     {
-        $exporterMock = $this->getExporterMockBuilder()->setMethods(array('getConfig'))->getMock();
-        $exporterMock->expects($this->any())->method('getConfig')->willReturn($this->getConfigMock());
+        $salesPricesMock = $this->getMockBuilder('\Findologic\Plentymarkets\Parser\SalesPrices')
+            ->disableOriginalConstructor()
+            ->setMethods(array('getDefaultPrice', 'getDefaultRrp'))
```

Comment:
Since you had to change a big portion of this test anyway, we could move from `array` to `[ ]`.